javascript/javascript3/week3/saveScreenshot.js

```diff
@@ -0,0 +1,53 @@
+const BASE_URL = `https://crudcrud.com/api/${API_KEY}`
+
+// Save Screenshot
+const saveScreenshotButton = document.querySelector(".saveScreenshot");
+
+const saveScreenshot = async (event) => {
+    event.preventDefault();
+    
+    const searchInput = document.querySelector("#url").value
```

Comment:
You still haven't validated that they have entered a valid URL (perhaps using regex)